packages/celotool/src/lib/generate_utils.ts

```diff
@@ -213,7 +215,8 @@ export const generateGenesisFromEnv = (enablePetersburg: boolean = true) => {
   const lookbackwindow = parseInt(fetchEnvOrFallback(envVar.LOOKBACK, '12'), 10)
   const chainId = parseInt(fetchEnv(envVar.NETWORK_ID), 10)
 
-  const initialAccounts = getFaucetedAccounts(mnemonic)
+  const initialAccounts = INITIAL_ACCOUNTS
```

Comment:
Is there a way we can keep this tool more general? As is, it would break deployments for all networks other than RC0

Reply:
Please review the new changes